bedrock/mozorg/templates/mozorg/incognito-browser.html

```diff
@@ -0,0 +1,199 @@
+      list, Tracking Protection blocks it.
+    {% endtrans %}
+    </p>
+
+    <h3 class="c-section-title">{{ _('Firefox Multi-Account Containers') }}</h3>
+
+    <p>
+    {% trans containers='https://addons.mozilla.org/firefox/addon/multi-account-containers/' %}
+      The Firefox <a href="{{ containers }}">Multi-Account Containers</a> add-on isn’t technically
+      a form of private browsing or tracking protection, but it can help keep companies from knowing
+      everything you do online. It lets you open fresh, cookie-free tabs that can be used for
+      different accounts—personal, work, shopping, etc. That means you can use Multi-Account
+      Containers to open several Google accounts at once without any overlap. Most trackers
+      won’t associate the different accounts, keeping your work life separate from your personal
+      life online. Some more advanced trackers, however, can and will track you across different
+      accounts, so beware.
+    {% endtrans %}
+    </p>
+
+    <h3 class="c-section-title">{{ _('Is Incognito/Private Mode Really Private?') }}</h3>
+
+    <p>
+    {% trans %}
+      Incognito or private mode will keep your local browsing private, but it won’t stop your ISP,
+      school, or employer from seeing where you’ve been online. In fact, your ISP has access to all
+      your browsing activity pretty much no matter what you do.
+    {% endtrans %}
+    </p>
+
+    <p>
+    {% trans vpn='https://blog.mozilla.org/internetcitizen/2017/08/29/do-you-need-a-vpn/' %}
+      You can, however, use a <a href="{{ vpn }}">Virtual Private Network (VPN) service</a>. VPN
+      services route traffic through remote servers, so it looks like you’re browsing from another
+      location or multiple locations. VPN providers can track where you’ve been online, though, so
+      it’s good to find a company you can trust to either delete or lock up your browsing activity.
+      VPNs won’t block third-party cookies from advertisers, but those cookies won’t be able to
+      identify your location accurately, making it difficult or impossible for ad trackers to be
+      effective.
+    {% endtrans %}
+    </p>
+
+    <p>
+    {% trans tor='https://www.torproject.org/' %}
+      <a href="{{ tor }}">Tor Browser</a> can truly mask your online activity. It bounces traffic
+      through multiple servers around the globe, making it difficult to track that traffic. The
+      website you visit really has no idea where you are, only the approximate location of the last
+      server your request was routed through. But again, even Tor proxy won’t stop third-party
+      advertisers from installing cookies in your browser. Tor Browser deletes all cookies when
+      closed. People can also start a new session in Tor Browser to clear them as well.
+    {% endtrans %}
+    </p>
+
+    <h3 class="c-section-title">{{ _('Incognito: Tldr') }}</h3>
```

Comment:
Nit: I think this is typically written as "tl;dr", with a semicolon, and for title case here it should even be "TL;DR" in all caps. I question using such meme culture jargon in this kind of copy at all, but that's not my call. But if we're going to do it we should get it right.

Also if there is any plan to localize this text we should add an explanatory 1l0n comment, maybe with an alternative for translation. Maybe "Incognito: The Short Version"?